Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
+    Those and their dependant targets will be generated as sources. \
+    If no target is specified, all the project (excluding the external ones) targets will be focused.
```

Comment:
```suggestion
    If no target is specified, all the project targets will be generated (except external ones, such as Swift packages).
```

`will be focused` does not sound right, so I tried to rephrase but feel free to change 🙂